### PR TITLE
Add Monthly Update for November 2022

### PR DESCRIPTION
**Pull request by lauragift21**
RubyGems update roundup for November 2022.

Hi @deivid-rodriguez, can you help review this month's update to ensure I accurately captured the work done in November? We also added a contributor appreciation section. Let me know if you have any feedback on it.

### CODE REVIEW

File: _posts/2022-12-12-november-rubygems-updates.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at Ruby Central, we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in November.
+
+## RubyGems News
+This month in RubyGems, we released new versions of RubyGems [3.3.25](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3325--2022-11-02), [3.3.26](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3326--2022-11-16) and Bundler [2.3.25](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2325-november-2-2022), [2.3.26](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2326-november-16-2022).
+
+The following improvements and fixes are also included in these releases (see the changelog for more information):
+
+- avoid installing everything under `lib` at the top in all test files - [#6074]( https://github.com/rubygems/rubygems/pull/6074).
```
Comment (deivid-rodriguez) [inline _posts/2022-12-12-november-rubygems-updates.md:15]: Let's change this to something summarizing better what that PR did, like

```suggestion
- improve some test times and fix warning by not installing documentation. - [#6074](https://github.com/rubygems/rubygems/pull/6074).
```

File: _posts/2022-12-12-november-rubygems-updates.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+- fixed broken link in `bundle-platform` man page - [#6071](https://github.com/rubygems/rubygems/pull/6071).
+- added permission restrictions to GitHub actions - [#6081](https://github.com/rubygems/rubygems/pull/6081).
+- removed reference to RVM documentation in the message returned when you run `bundler outdated` - [#6083](https://github.com/rubygems/rubygems/pull/6083).
+
+```bash
+$ bundle outdated
+Could not load OpenSSL.
+You must recompile Ruby with OpenSSL support or change the sources in your Gemfile from 'https' to 'http'. Instructions for compiling with OpenSSL using
+RVM are available at rvm.io/packages/openssl.
+```
```
Comment (deivid-rodriguez) [inline _posts/2022-12-12-november-rubygems-updates.md:25]: I don't think we need all the above, the one liner with the link seems fine.

File: _posts/2022-12-12-november-rubygems-updates.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+- added a specification to ensure that global gemspecs do not confuse Bundler - [#6086](https://github.com/rubygems/rubygems/pull/6086).
```
Comment (deivid-rodriguez) [inline _posts/2022-12-12-november-rubygems-updates.md:26]: We have a concept on "specification" inside our code, so I think using "test" reduces potential confusion

```suggestion
- added a test to ensure that global gemspecs do not confuse Bundler - [#6086](https://github.com/rubygems/rubygems/pull/6086).
```

We

File: _posts/2022-12-12-november-rubygems-updates.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+- fixed issue that occurs when a lockfile gem does not resolve on the current platform - [#6070](https://github.com/rubygems/rubygems/pull/6070).
+- updated the docs for `gemfile` man page - [#6007](https://github.com/rubygems/rubygems/pull/6007).
+
+![Gemfile Man](https://i.imgur.com/veOa0z4.png)
+
```
Comment (deivid-rodriguez) [inline _posts/2022-12-12-november-rubygems-updates.md:31]: I'd say we don't need the picture here either, the one liner with a link to the PR seems fine.

**Review by deivid-rodriguez — COMMENTED** [review]
Minor tweaks on wording but looks good other than that, thanks for doing this!

File: _posts/2022-12-12-november-rubygems-updates.md (hunk 4 above)
Comment (lauragift21) [reply _posts/2022-12-12-november-rubygems-updates.md:31]: Yes, this was suggested by Irene. We wanted to highlight updates with screenshots or images of what changed. Assuming this doesn't fit, I'll move them to a different section linking back to the one-liner.

**Pull request merged**